pkg/apis/migration/v1alpha1/migcluster_types.go

```diff
@@ -175,6 +176,25 @@ func (m *MigCluster) GetRsyncTransferImage(c k8sclient.Client) (string, error) {
 	return rsyncImage, nil
 }
 
+// GetClusterSubdomain gets a MigCluster specific subdomain value to be used for DVM routes
+func (m *MigCluster) GetClusterSubdomain(c k8sclient.Client) (string, error) {
+	client, err := m.GetClient(c)
+	if err != nil {
+		return "", err
+	}
+	clusterConfig := &corev1.ConfigMap{}
+	clusterConfigRef := types.NamespacedName{Name: ClusterConfigMapName, Namespace: VeleroNamespace}
+	err = client.Get(context.TODO(), clusterConfigRef, clusterConfig)
+	if err != nil {
+		return "", liberr.Wrap(err)
+	}
+	clusterSubdomain, ok := clusterConfig.Data[ClusterSubdomainKey]
+	if !ok {
+		return "", liberr.Wrap(errors.Errorf("configmap key not found: %v", ClusterSubdomainKey))
```

Comment:
For anyone else reading this about to suggest we not return an err here, the err is ignored so it's OK